src/TermStoreSchemaUpdater.php

```diff
@@ -8,6 +8,10 @@ class TermStoreSchemaUpdater {
 
```

Comment:
Changes to this file (and related change in MediaWikiNormalizedTermCleanerTest.php) were not necessary .. they slipped through when I rebased .. but I'll avoid creating a commit to revert them unless reviewers would prefer that